packages/qdrant-loader-core/src/qdrant_loader_core/llm/providers/bedrock.py

```
@@ -0,0 +1,299 @@
+            payload = json.loads(raw_text)
+            embeddings = _extract_embeddings(payload)
+
+            if self._expected_vector_size is not None:
+                for vector in embeddings:
+                    if len(vector) != self._expected_vector_size:
+                        raise ServerError(
+                            "Bedrock returned embedding vector with unexpected dimension"
+                        )
+
+            return embeddings
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)